### ISSUE TITLE
「グラフ」ページで「追加」ボタンを押した時にページ遷移確認ダイアログが出る

### ISSUE THREAD

**Issue by Mnakagawa** [project-hatohol/hatohol#2235]
@mmikan22 さん
タイトルのとおりなのですが，「グラフ」ページで「追加」ボタンを押した時にページ遷移確認ダイアログが出ます。
URLが
http://hoge/ajax_graph -> http://hoge/ajax_history?mode=createと変わるからなのでしょうか？

**Comment by Mnakagawa** [comment]
http://hoge/ajax_history?mode=create
にて，「閉じる」を押し，http://hoge/ajax_graphに戻るときにも同様に発生する

**Comment by kz0817** [comment]
これ、修正漏れですね。直してPRします。

**Comment by Mnakagawa** [comment]
「最新データ」ページにおいても，「ページ毎のアイテム数」を変更し，エンターを押した段階で確認ダイアログが出る。

**Comment by kz0817** [comment]
> 「最新データ」ページにおいても，「ページ毎のアイテム数」を変更し，エンターを押した段階で確認ダイアログが出る。

ここも漏れですね。修正します。

**Comment by kz0817** [comment]
> 「最新データ」ページにおいても，「ページ毎のアイテム数」を変更し，エンターを押した段階で確認ダイアログが出る。

よく観察すると、指定したレコード数にデータが更新されたあと、refreshが行われているように思える。なぜ？

**Comment by kz0817** [comment]
> よく観察すると、指定したレコード数にデータが更新されたあと、refreshが行われているように思える。なぜ？

いや、FFとChromeでここらへんが動きが微妙に違う。最終的には同じになっているが。

**Comment by kz0817** [comment]
4.10.22.2 Implicit submissionが働いている？
ダミーのtextフィールドをformに追加すると、問題が再現しなくなる。

<pre>
diff --git a/client/viewer/latest_ajax.html b/client/viewer/latest_ajax.html
index 5c43526..8461953 100644
--- a/client/viewer/latest_ajax.html
+++ b/client/viewer/latest_ajax.html

@@ -61,6 +61,7 @@
       <option>_non_</option>
     </select>
     <label for="num-records-per-page">{% trans "# of items per page" %}</label>
+    <input type="text" name="dummy" style="display:none;">
     <input type="text" id="num-records-per-page" class="form-control num-records-per-page" style="width:4em;">
   </form>
<pre>

**Comment by kz0817** [comment]
#2237のPRにこの問題の解決用パッチも追加しました。

**Comment by Mnakagawa** [comment]
ざっくり触った感じ，問題なさそうなのでクローズ。再発したらRe-Openする。
